src/__experimental__/components/search/search.d.ts

```diff
@@ -9,13 +9,17 @@ export interface SearchProps {
    * `onClick` events are triggered when the `searchButton` is clicked
    */
   onClick?: (ev: React.SyntheticEvent) => void;
+    /** Prop for `onMouseDown` events.
```

Comment:
This is why your linter has failed fyi, I also think we don't need to surface this prop

Reply:
It's gone on the next commit.